packages/integration-tests/src/config.ts

```diff
@@ -0,0 +1,20 @@
+import { Wallets } from "./entities";
+
+export const localConfig = {
+  gasLimit: { gasLimit: 10000000 },
```

Comment:
What is this? Is it for L1? Should it be `l1GasLimit: 10000000` then?

Reply:
updated